R/modify_columns.R

```diff
@@ -714,6 +733,9 @@ cols_move <- function(data,
 cols_hide <- function(data,
                       columns) {
 
+  # Perform input object validation
+  # stop_if_not_gt(data = data)
```

Comment:
?

Reply:
With https://github.com/rstudio/gt/pull/447, this can be uncommented. (I realize now that this review should have waited until that PR is merged.)

This reason this check can't occur now is that `cols_hide()` is used once in the initialization of the object (before the object is given the `gt_tbl` class). With https://github.com/rstudio/gt/pull/447, that call disappears and the above can be uncommented and a test can be added for `cols_hide()`.

Reply:
Will uncomment this in https://github.com/rstudio/gt/pull/447 after merging.